### PR TITLE
Clean up unused styles

### PR DESCRIPTION
**Pull request by emandilara**
Cleaning up some CSS classes that do not have an implementation or are unused.

### CODE REVIEW

File: kitchensink/src/app/static/foundations/IconsPage.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const IconsPage = () => (
           Two-tone and multi-color icons are not available in the icon font. These icons are available as SVG assets.
         </DescriptionBox>
 
-        <div className="iconGrid">
```
Comment (cyrilwanner) [inline kitchensink/src/app/static/foundations/IconsPage.jsx:35]: Are you sure this is not used anymore?
On https://open-ch.github.io/pyrene/icons it looks like this class still adds some styles (looks like they are added globally here: https://github.com/open-ch/pyrene/blob/3ba1d1e37ebf2a7d6e249d38f632961246c716ba/kitchensink/src/css/common.css#L121) and the page looks kind of broken when I remove the class.

**Review by cyrilwanner — CHANGES_REQUESTED** [review]
I think there is one issue with the icons page, but otherwise, it looks good to me.
Thanks for cleaning them up!

File: kitchensink/src/app/common/Search/ResultsPage.jsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export default class ResultsPage extends React.Component {
       <div className={styles.header}>
```
Comment (sanmiawi) [inline kitchensink/src/app/common/Search/ResultsPage.jsx:40]: I guess this style can also be removed

File: kitchensink/src/app/static/foundations/IconsPage.jsx (hunk 1 above)
Comment (emandilara) [reply kitchensink/src/app/static/foundations/IconsPage.jsx:35]: Indeed, thank you for pointing that out!

**Review by cyrilwanner — APPROVED** [review]

**Pull request merged**